elastio-aws-resources-tagging/README.md

```diff
@@ -0,0 +1,96 @@
+In certain AWS accounts, custom tags are necessary for resource management. To streamline and automate this process, we have designed a script for tagging elastio AWS resources.
+
+## Requirements
+Please ensure that you have the following prerequisites in place:
+ - Linux box with the AWS Command Line Interface (CLI) and the jq tool installed. 
+ - Ensure that you have established connection to the AWS account where elastio is currently running.
+
+OR
+
+Alternatively, you can use a cloud shell console directly within the AWS account where elastio is operational. 
+
+### Script usage
+
+Create a new `script.sh` file and copy the content of the script to the file.
+
+Replace keys and values with your desired custom tags:
+
+```
+tags='{"Tags":[{"Key": "name1","Value": "value2"},{"Key": "name2","Value": "value2"}]}'
```

Comment:
It's unusual for a script to work by requiring modification of its code.  The AWS CLI itself takes tags as a command line argument.  Can't we do the same?  Something like:

`./foo.sh tag1=value1 tag2=value2 tag3 tag4 tag5=value5`

If you're not sure how to parse such command line args in a shell script, I'm sure ChatGPT can generate some initial code you can tweak.  LMK if you need help w/ this.